Dockerfile

```diff
@@ -16,14 +16,22 @@ LABEL name="Httpd" \
       vendor="ManageIQ" \
       description="Apache HTTP Server"
 
-RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
-    dnf -y --disableplugin=subscription-manager module enable mod_auth_openidc && \
-    dnf config-manager --setopt=appstream*.exclude=*httpd* --save && \
+RUN if [ $(uname -m) != "s390x" ] ; then \
+      dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
+        http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
+        http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
+      dnf config-manager --setopt=appstream*.exclude=*httpd* --save && \
+      dnf -y --disableplugin=subscription-manager module enable mod_auth_openidc && \
+      dnf -y --disableplugin=subscription-manager install mod_auth_openidc; \
+    else \
+      dnf -y install \
+        /opt/app-root/src/bin-rpm-dir/cjose-0.6*.s390x.rpm \
```

Comment:
Do we need to add the version numbers here?

Reply:
This was sort of discussed in the related postgres PR: https://github.com/ManageIQ/container-postgresql/pull/39#discussion_r837566634